### PR TITLE
Fix podcast promo 500s

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves [No Issue]

**Overall change:**
Fix a 500 caused when the inline podcast promo is rendered on a service that does not have `skipLink` translations

**Code changes:**

- _Remove the possibility of null reference exception._
- _Update tests to be less brittle to service config changes._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by rebeccamcginn — APPROVED** [review]

File: src/app/containers/PodcastPromo/Inline.jsx [hunk 1]
```diff
@@ -144,6 +144,13 @@ const StyledCardLink = styled(PromoComponent.Card.Link)`
   display: block;
 `;
 
+const buildSkipLinkComponent = fields => props =>
+  Object.values(fields).every(Boolean) ? (
+    <SkipLinkWrapper {...fields} {...props} />
+  ) : (
+    <Fragment {...props} />
```
Comment (andrewscfc) [inline src/app/containers/PodcastPromo/Inline.jsx]: Does this just not render a skip link when translations aren't available? Should we just not render the podcast promo as the promo will be inaccessible to screen users with this approach? It's also less code and we should get some translations anyway?

Comment (ryanmccombe) [reply src/app/containers/PodcastPromo/Inline.jsx]: Aye - removes the skip button if we don't have translations for it.  Content would be even less accessible if it wasn't on the page at all 😛 will check with product though

Comment (andrewscfc) [reply src/app/containers/PodcastPromo/Inline.jsx]: >  Content would be even less accessible if it wasn't on the page at all 

Yeah true.. Just wonder if we should just not show it until we get some translations, surely they won't take long to get?

Anyway like you say product call I guess

**Review by MuhammadAHussain — APPROVED** [review]
LGTM 👍

File: src/app/containers/PodcastPromo/Inline.jsx (hunk 1 above)
Comment (ryanmccombe) [reply src/app/containers/PodcastPromo/Inline.jsx]: Product call was neither!  It was to use English instead

**Pull request merged**